pkg/drivers/kic/oci/oci.go

```diff
@@ -69,6 +69,25 @@ func DeleteContainersByLabel(ociBin string, label string) []error {
 	return deleteErrs
 }
 
+// DeleteContainer deletes a container by ID or Name
+func DeleteContainer(ociBin string, name string) error {
+	if err := PointToHostDockerDaemon(); err != nil {
+		return errors.Wrap(err, "point host docker daemon")
+	}
+	_, err := ContainerStatus(ociBin, name)
+	// only try to delete if docker/podman inspect returns
+	// if it doesn't it means docker daemon is stuck and needs restart
+	if err != nil {
+		glog.Errorf("%s daemon seems to be stuck. Please try restarting your %s. Will try to delete anyways", ociBin, ociBin)
```

Comment:
Please include the error message seen, for debugging sake.

There may be other reasons that ContainerStatus returns an error other than being stucj.

Reply:
good point